Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_chamber.nut

```diff
@@ -11,15 +11,9 @@ void function GamemodeChamber_Init()
 	Riff_ForceBoostAvailability( eBoostAvailability.Disabled )
 	ClassicMP_ForceDisableEpilogue( true )
 
-	AddCallback_OnClientConnected( ChamberInitPlayer )
 	AddCallback_OnPlayerKilled( ChamberOnPlayerKilled )
-	AddCallback_OnPlayerRespawned( UpdateLoadout )
-
-}
-
-void function ChamberInitPlayer( entity player )
-{
-	UpdateLoadout( player )
+	//AddCallback_OnPlayerRespawned( UpdateLoadout ) // should never set up loadout in respawn function

```

Comment:
@F1F7Y we probably don't want leftover commented out code in codebase, right?

Reply:
If it's documented with a comment id say it can stay. My thinking was that one day:tm: we'll format this so we don't really need to push for perfection right now

Reply:
I mean, it's not really _perfection_ but just proper cleanup before merge :P